### PR TITLE
Milestone 6: Accessibility 

### PR DESCRIPTION
**Pull request by ezekielgn**
In this milestone, my coding partner and I improved the accessibility of the website by giving the alt attributes of the images a more meaningful name and also adding labels for each input.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,255 +1,373 @@
...
-				</form>
-			  </div>
-			</div>
-		</section>
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
```
Comment (GedeonTS) [inline index.html]: - Kindly make sure your name is in the title as it is a good practice for devs portfolio :recycle:

File: index.html [hunk 2]
```diff
@@ -1,255 +1,373 @@
...
+        <h1 class="title1">I'm Gardimy. <br />Glad to see you!</h1>
+        <p class="title2">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+
+        <div class="social-icon">
+          <h2 class="connect" href="#">LET'S CONNECT</h2>
```
Comment (GedeonTS) [inline index.html:37]: - I noticed that your website still has some contrast issues kindly fix them:
Screenshot from 2023-04-04 20-12-00
Screenshot from 2023-04-04 20-13-02
__Hint: You can use this [link](https://color.a11y.com/?wc3) to test your contrast ratio.__

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi team @PabloEGonz and @Gardimy , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 
## Required Changes ♻️
-[ ]  As per the project requirements [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md#:~:text=Each%20point%20on,request%20changes.), kindly mention all listed aspects of accessibility in your PR message and write "no issues found" if it's ok otherwise solve it and add a link to the modified code.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @PabloEGonz and @Gardimy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text

## Highlights
Used GitHub flow ✅
Used linters ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**